lib/PreconditionRunner.pm

```diff
@@ -95,12 +96,53 @@ sub write_num_passes
 sub run_to_steady_state($)
 {
     my $self = shift;
-    
     my %args = @_;
 
     my $msg_prefix = $args{'msg_prefix'};
-    my $output_dir = $args{'output_dir'};
     my $test_ref = $args{'test_ref'};
+    my $output_dir = $args{'output_dir'};
+
+    # preconditioner.exe does not support multiple files
```

Comment:
ooo, I like that idea.  You're correct -- as-is the code just runs diskspd for an hour with the same XML profile that I use for the test.  To implement your preconditioner-based solution, StorScore needs to understand each workload -- information which right now is locked in the XML file.  I might need to wait for the next change, when I work out how the recipe should describe the workload for each "sub target."